lib/code_ownership/private/parse_js_packages.rb

```diff
@@ -32,6 +32,14 @@ def self.from(pathname)
             name: package_name,
             metadata: package_loaded_json[METADATA] || {}
           )
+        rescue JSON::ParserError
```

Comment:
What is the message from this? Might be useful to include in the message.

Reply:
```
/home/ruby/apps/my_app/vendor/bundle/ruby/2.7.0/gems/json-2.3.1/lib/json/common.rb:263:in `parse': 784: unexpected token at '{ (JSON::ParserError)
```

(from the issue above).

Reply:
Cool, so maybe add `=> e` to the rescue, and include #{e.message} on the first line?

Reply:
Done!